app/index.html

```diff
@@ -13,7 +14,41 @@ <h1>HELLO WORLD!</h1><i class="fa fa-hand-spock-o" aria-hidden="true"></i>
   	<div class="header header__second"></div>
   	<div class="header header__third"></div>
   	<div class="header header__fourth"></div>
-  	<div class="header header__fifth"></div>
+  	<div class="header header__fifth">
+		<main class="fifth">
+			<nav class="fifth__menu">
+				<img alt="Logo UI" src="img/logo.png" class="fifth__menu--logo">
+				<ul class="fifth__navigation">
+					<li class="fifth__navigation--item"><a href="#" >PRODUCTS</a></li>
```

Comment:
please, don't use CAPS. You can make text uppercased with css `text-transform: uppercase`

Reply:
Corrected.